server/controllers/medical/patients/groups.js

```diff
@@ -114,3 +114,39 @@ function update(req, res, next) {
     .catch(next)
     .done();
 }
+
+// assign multiple patient to a group
+function bulkUpdate(req, res, next) {
+
+  const { patientUuids, subscribedGroups } = req.body;
+
+  const uuids = [].concat(patientUuids);
+  const groups = [].concat(subscribedGroups);
+
+  // Clear assigned groups
+  const removeAssignmentsQuery = 'DELETE FROM patient_assignment WHERE patient_uuid = ?';
```

Comment:
I mentioned this to you in person, but it would be great to add an option to skip the "delete" request on the client.  This way would could support both:
 1. Replace all groups by deleting the old values OR
 2. Appending new values to the patients group list